tests/sentry/migrations/test_0729_backfill_groupsearchviews_with_pinned_searches.py

```diff
@@ -19,6 +21,7 @@ def setup_initial_state(self):
             sort="date",
         )
 
+    @pytest.mark.skip(reason="old migration test")
```

Comment:
If these tests don't work anymore, why not delete them?

Reply:
I think it's helpful to keep some around as an example, but we could delete old skipped tests after a while